save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -65,8 +72,15 @@ class WarnPlugin(testConfig: TestConfig) : Plugin(testConfig) {
                 }
             }
             .mapValues { it.value.sortedBy { it.message } }
-        // todo: create a temp file with technical comments removed and feed it to the tool
-        val executionResult = pb.exec(warnPluginConfig.execCmd, null)
+
+        var execCmd: String = warnPluginConfig.execCmd
```

Comment:
We have discussed, that we should append file name to the cmd anyway (like we discussed for #88). I think we should do it here too